packages/browser/src/integrations/linkederrors.ts

```diff
@@ -48,7 +49,7 @@ export class LinkedErrors implements Integration {
    */
   public setupOnce(): void {
     addGlobalEventProcessor((event: Event, hint?: EventHint) => {
-      const self = getCurrentHub().getIntegration(LinkedErrors);
+      const self = getIntegration(getCurrentHub(), LinkedErrors);
       return self ? _handler(self._key, self._limit, event, hint) : event;
```

Comment:
`self` is no longer this ... therefore, `_key` and `_limit` should be `public` ... thinking later how to continue to do more FP tuff in another part of the code base